contracts/test/WETH9.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.0;
+
+import '../interfaces/external/IWETH.sol';
+
+contract WETH9 is IWETH {
+    string public name     = "Wrapped Ether";
+    string public symbol   = "WETH";
+    uint8  public decimals = 18;
+
+    mapping (address => uint)                       public  balance;
+    mapping (address => mapping (address => uint))  public  allowances;
+
+    receive() external payable {
+        deposit();
+    }
+    
+    function deposit() public payable override {
+        balance[msg.sender] += msg.value;
+        emit Deposit(msg.sender, msg.value);
+    } 
+
+    function withdraw(uint amount) public override {
+        require(balance[msg.sender] >= amount);
+        balance[msg.sender] -= amount;
+        payable(msg.sender).transfer(amount);
+        emit Withdrawal(msg.sender, amount);
+    }
+
+    function balanceOf(address account) public view override returns (uint256) {
+        return balance[account];
+    }
+
+    function allowance(address owner, address spender) public view override returns (uint256) {
```

Comment:
get rid of this. Look at comment about allowances mapping

Reply:
done